test/registries.test.js

```diff
@@ -4,6 +4,16 @@
 // License text available at https://opensource.org/licenses/MIT
 
 describe('Registry', function() {
+  describe('models', function() {
+    it('should throw error upon extending non-exist base model', function(done) {
+      var app = loopback();
+      var props = {};
+      var opts = { base: 'nonexistent' };
+      expect(function() { app.registry.createModel('aModel', props, opts); })
+        .to.throw();
```

Comment:
Please check that the error message contains both the name of the model being created (`aModel`) and the non-existing base model (`nonexistent`).